Product card

Beepli opened the pull request:
- [DEMO LINK](https://Beepli.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://Beepli.github.io/layout_product-cards/report/html_report/)

volodymyr-soltys97 commented on src/index.html:
```diff
@@ -1,18 +1,84 @@
+          5K Monoblock (MNED2UA/A)
+        </h1>
+
+        <h2 class="header__product">Product code: 195434</h2>
```
Consider will be better if you use the tag p instead of h2, fix it everywher
```suggestion
        <p class="header__product">Product code: 195434</p>
```

volodymyr-soltys97 commented on src/index.html:
```diff
@@ -1,18 +1,84 @@
+    </header>
+
+    <div class="review">
+      <div class="stars-wrapper">
```
Redundant
```suggestion
```

volodymyr-soltys97 commented on src/index.html:
```diff
@@ -1,18 +1,84 @@
+      <h3 class="price__actual">$2,199</h3>
+    </div>
+
+    <div class="bottom">
```
Redundant
```suggestion
```

volodymyr-soltys97 commented on src/index.html:
```diff
@@ -1,18 +1,84 @@
+    </div>
+
+    <div class="bottom">
+      <div
+        class="bottom__button"
+        data-qa="hover"
+      >
+        BUY
+      </div>
```
Use the semantic tag for the button

volodymyr-soltys97 commented on src/styles/style.scss:
```diff
@@ -0,0 +1,140 @@
+.page {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-style: normal;
```
These values are default
```suggestion
```

lerastarynets replied:
not fixed

volodymyr-soltys97 commented on src/styles/style.scss:
```diff
@@ -0,0 +1,140 @@
+  font-size: 10px;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-position: center;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-image: url(../images/star.svg);
+}
+
+.stars__star.is-active {
+  background-image: url(../images/star.svg);
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
+}
```
Move stars styles to the separate file

lerastarynets replied:
not fixed

etojeDenys replied:
<img width="690" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/70220667/908818e9-6d8d-4351-8e20-87437df3c38e">

volodymyr-soltys97 reviewed (changes requested):
Good job 👍 
Let's make your code better.
You need to fix the failed test, for example you can add padding for the card container and remove the horizontal padding everywhere, also fix the button width
Also this part of the task is incomplete, you need to use styles from the previous task for stars block 
<img width="566" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/64750958/126b9fdd-631a-4ca7-8555-b69479a0b2de">

lerastarynets reviewed (changes requested):
pls fix all prev comments

etojeDenys reviewed (changes requested):
feel free to ask for some help in the chat

Beepli commented:
hi @etojeDenys 
I've already moved stars to stars.scss
and added imort @import 'blocks/stars';
I pushed deploy again
Could you please check and let me know if changes are visible

lerastarynets reviewed (approved):
lgtm 🚀
